.github/workflows/test.yml

```diff
@@ -0,0 +1,61 @@
+name: Test code functionality
+
+on:
+  push:
+    branches:
+      - '**'
+    tags:
+      - '**'
+  pull_request:
+    branches:
+      - '**'
+
+jobs:
+  tests:
+    runs-on: ubuntu-22.04
+    timeout-minutes: 15
+
+    services:
+      postgres:
+        image: postgres
+        env:
+          POSTGRES_USER: postgres
+          POSTGRES_PASSWORD: secret
+          POSTGRES_DB: topobank-test
+        ports:
+          - 5432:5432
+
+    env:
+      USE_DOCKER: no
+      DJANGO_SETTINGS_MODULE: config.settings.test
+      DATABASE_URL: postgres://postgres:secret@localhost/topobank-test
+
+    strategy:
+      matrix:
+        python-version: ['3.10']
+
+    steps:
+    - uses: actions/checkout@v3
+      with:
+        fetch-depth: 0
+        submodules: recursive
+
+    - name: Set up python3 ${{ matrix.python-version }}
+      uses: actions/setup-python@v4
+      with:
+        python-version: ${{ matrix.python-version }}
+
+    - name: Install
+      run: |
+        pip install --upgrade pip
+        pushd /tmp
+        git clone --branch 23_tweaks2 https://github.com/ContactEngineering/topobank.git
```

Comment:
Same  question here as for `topobank_statistics` .. which `topobank` branch should be used here in general? (For now `23_tweaks2` is okay of course)

Reply:
I would wait with merging until the main topobank `branch` is updated. The tests are not yet running, I'll work on it.